training-server.yml

```diff
@@ -1,93 +1,156 @@
-# Production tuning of server limits.conf, Postgres and NGINX required, but this will install a minimal OMERO server,
-# OMERO.web with NGINX and systemd configuration controlling both.
+# Install OMERO.server and OMERO.web with a public user on localhost
 
-# Install the latest release, including PostgreSQL on the same server
+- hosts: outreach.openmicroscopy.org
+  pre_tasks:
```

Comment:
Minor point, but do these need to be `pre_tasks` (implying they're prerequisites for roles) instead of `post_tasks`?